odoo/addons/base/models/res_partner.py

```diff
@@ -48,7 +48,7 @@ class FormatAddressMixin(models.AbstractModel):
     def _fields_view_get_address(self, arch):
         # consider the country of the user, not the country of the partner we want to display
         address_view_id = self.env.company.country_id.address_view_id
-        if address_view_id and not self._context.get('no_address_format'):
+        if address_view_id and not self._context.get('no_address_format') and (not address_view_id.model or address_view_id.model == self._name):
```

Comment:
It's equal to ``and self._name == "res.partner"``, because   ``address_view_id.model`` is always ``res.partner``: 

https://github.com/odoo/odoo/blob/3b1df8f57e0cd342fe7f9ff7bd34100b2c821569/odoo/addons/base/models/res_country.py#L50-L52

And that means that you disable it for crm. No feature no bugs 
``¯\_(ツ)_/¯``

Reply:
I agree but It seems too easy :p

Reply:
Is it temporary solution? If not, why not to remove address view from crm?

https://github.com/odoo/odoo/blob/2eecff58239868ef596841534f4b5cffaa1d75c0/addons/crm/models/crm_lead.py#L663-L664
https://github.com/odoo/odoo/blob/2eecff58239868ef596841534f4b5cffaa1d75c0/addons/crm/models/crm_lead.py#L84

Reply:
Because this is a stable version and we keep the diff as minimal as possible. Also, this feature could be used using a `crm.lead` template on the country. This feature should be improved in master.